packages/providers/email-mailgun/README.md

```diff
@@ -44,9 +46,9 @@ module.exports = ({ env }) => ({
     config: {
       provider: 'mailgun',
       providerOptions: {
-        apiKey: env('MAILGUN_API_KEY'),
-        domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
-        host: env('MAILGUN_HOST', 'api.mailgun.net'), //Optional. If domain region is Europe use 'api.eu.mailgun.net'
+        key: env('MAILGUN_API_KEY'), // Required
```

Comment:
These aren't breaking changes but they would need to be added to a migration guide since the key names changed.

We will have to push this either to 4.1.11 or 4.2.x since we won't have time to review and get the migration guide built for the release in a few hours.

Reply:
Ignore this, there is a conversion function for the old keys to the new ones